src/client/components/Comments/Comments.js

```diff
@@ -210,6 +210,8 @@ class Comments extends React.Component {
     const rootLinkedComment = findRootComment(comments, getLinkedComment(comments));
     const commentsToRender = this.commentsToRender(rootLevelComments, rootLinkedComment);
 
+    const empty = !loading && commentsToRender.length === 0;
```

Comment:
Actually we dont need to load comments if the root post show that there is `comment_count: 0`. We can show this empty comment message directly without waiting for a loading.